docs/ENV.md

```diff
@@ -6,6 +6,7 @@ The following environment variables are available to configure the bom generatio
 | ------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | CDXGEN_DEBUG_MODE                           | Set to `debug` to enable debug messages                                                                                                                                                                                                                                                                                             |
 | GITHUB_TOKEN                                | Specify GitHub token to prevent traffic shaping while querying license and repo information                                                                                                                                                                                                                                         |
+| MULTI_BOM_COMPONENT_REF                     | When building a multi-language BOM, choose how the references between the components is handled. This can be useful if other tooling used to eg visualize the dependency-tree only uses 'dependsOn' (as the name dependency-tree more or less implies). Possible values: `dependsOn`, `provides`. Default: `provides`.              |
```

Comment:
Does this work with our configuration files? .cdxgenrc etc?

Reply:
Don't know, I didn't know there are config files! 😄 I'll try it and let you know what I find/try to fix it if it doen't.

Reply:
@prabhu Is it currently possible to put EnvVars in the config files? I can't seem to find any documentation on that, or in yargs for that matter... Or is this a feature that needs to be added?

Reply:
Good point. I think only command line arguments can be passed via [config](https://cyclonedx.github.io/cdxgen/#/ADVANCED?id=configuration-files) files. Do we need this variable or can we just assume `dependsOn` ?

Reply:
There's already lots of EnvVar-configs, so we could maybe leave it like that? Or add it as a cli argument.
Maybe it would be good to clean that up a bit at some point and have both support all current args and EnvVars?

Reply:
I meant, do we need this new variable `MULTI_BOM_COMPONENT_REF` at all?

Reply:
Not necessarily, I just thought it would be good the have the option to choose. I initially didn't have it, then decided to change the default to `provides`, since this made more sense to me.
I'll remove it and set it to `dependsOn`; We can always add something if a request is made. 😄